sql/analyzer/resolve_tables.go

```diff
@@ -44,11 +44,14 @@ func resolveTables(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, sel
 	span, _ := ctx.Span("resolve_tables")
 	defer span.Finish()
 
+	createTriggerIgnore := false
 	return transform.NodeWithCtx(n, nil, func(c transform.Context) (sql.Node, transform.TreeIdentity, error) {
 		ignore := false
 		switch p := c.Parent.(type) {
 		case *plan.DropTable:
 			ignore = p.IfExists()
+		case *plan.CreateTrigger:
+			createTriggerIgnore = true
```

Comment:
(minor) You could just re-use `ignore` to simplify just a little bit – one less variable to track and it seems like `ignore` would already capture the essence of what you want to do.

Reply:
a short comment here wouldn't hurt either